scio-core/src/main/scala/com/spotify/scio/runners/dataflow/DataflowResult.scala

```diff
@@ -109,10 +109,10 @@ object DataflowResult {
     new DataflowResult(internal)
   }
 
-  private def getOptions(projectId: String): DataflowPipelineOptions = {
-    val options =
-      PipelineOptionsFactory.create().as(classOf[DataflowPipelineOptions])
+  private def getOptions(projectId: String, region: String): DataflowPipelineOptions = {
+    val options = PipelineOptionsFactory.create().as(classOf[DataflowPipelineOptions])
     options.setProject(projectId)
+    options.setRegion(region)
```

Comment:
@danielnorberg that option is still marked as `@Experimental` given that, I kinda tend to prefer the behavior as is ... you specifically set the region.

@nevillelyh WDYT?

Reply:
I’m fine with whatever as long as it is possible to pass in the region to `DataflowResult.apply()` and get results for jobs that are outside `us-central1`

Reply:
@regadas would you prefer this https://github.com/spotify/scio/pull/1480 ?

Reply:
Note that unless we set the region on the `DataflowPipelineOptions`, calling `getJob` on the `DataflowResult` will not work because it uses the region of the `DataflowPipelineOptions` when querying for the dataflow job.

Reply:
@danielnorberg yeah I had a second look at this and it seems this is the best approach, plus they use this option all over the place in `apache-beam` `DataflowRunner` and `DataflowClient`.